### PR TITLE
Add the welcome image

### PR DESCRIPTION
**Pull request by Johan-Ng**
Please check the changes made. Thank you!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
...
...HG5jmytLHOEwa06PZlF3c2kg8ra5LVO2iopS9rHWXgBYLB9QBAEB8ugPE7C5pAcWkggOFri4tcX0uPFAaXh/EpXiSnnLRVQWa820e036uoa38L7ajk4OHJASOGKesjgDa2Zk02Z3bYMoy37I2Gvp890BtkAQBAEAQBAEAQBAEAQBAEAQBAEAQBARMVxKKmidLK7K1tuRJJJsGtaNXOJNgBqUBWKAOxZnWyuDKMkgUzXdt5abEVTh7Oo1hHxE7ID1V4jJUDqsOAbHH2HVNvumgaFkDRYSvG1x2G95IyrnbQpLT02/2eSeM9/YvPBLSe/SSMHw1sALIy8ue7M97nFz3vIAL3u5mwA7gAALALzruLi4qKKk8vckm0kWtEYIssTLABeuo0+jgoZzjmyjJ5eSLU0he+97CwU6lhHPr2sqtTOcIzQ0rG8te8rDk2T0rWnT3pb+tmrxDALymenmdTzOtnc0NcySwsOtido4gbOBDtLXtosFgwYFwyIZ3VMzo5J3NyZo4Wwta29zZgc4lxNruLidBsEBYkAQBAYKqoDB4nYLKWSvcV1SjnnyNY2ocHZr6/8ANFJjcceNxUjPpM7za08wcLhRtYO3RqqrHUjT8S4bISyppx/iYM2UbdbE63WU7j3OsCDyc1p774JTY4TiUdTCyaI3Y8XFxYg7FrhycCCCORBQExAEAQBAEAQBAEAQBAEAQBAEAQBAEAQGs4hwx1REGxvDJGPjljcRcB8bg4Bw5tOrTzsTbVAV6m4WqJ5ppKwsiim6syUsD3OZK9jcpfLIWscWkWBYBqGtzE7IC4RQta0Na0NaAAGgAAAaAADYLDSe5gw01JlcT8vJcyz2crerKf8A89i/N3cTTq6kkSl1CEIAgCAIAgCAIDxLEHCxWU8EdSnGpHTI1zaF2ax27+//AHW+rccuNjPpNL4dZsmMAFgozrRiorC4HpDYhUGGxwvldGCOuf1jm3OXPYNLmt90mwJtuddyUBNQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEB//2Q==" />
```
Comment (itaditya) [inline src/index.html]: why are you using blob for image?

**Comment by Johan-Ng** [comment]
I tried the link but, it doesn't seem to work. The image wasn't shown. I'll
try to give that another go. Thank you so much for your guidance!

File: src/index.html (hunk 1 above)
Comment (Johan-Ng) [reply src/index.html]: Please have a look when you are free! Thank you!

**Comment by itaditya** [comment]
@Johan-Ng sorry but there is no task opened which says add a Welcome image. Thanks for working on it but this is not a contribution I want.

**Pull request closed without merging**